.github/workflows/ci.yaml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version:
+        - '2.6'
+        - '2.7'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
+        bundler-cache: true
+    - name: Run tests
+      run: bundle exec rake
+  build:
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up environment
+      run: |
+        echo "TRAVIS_PULL_REQUEST=${{ github.base_ref == '' && 'false' || 'true' }}" >> $GITHUB_ENV
+        echo "REGISTRY_PASSWORD=${{ secrets.REGISTRY_PASSWORD }}" >> $GITHUB_ENV
+        echo "REGISTRY_USERNAME=${{ secrets.REGISTRY_USERNAME }}" >> $GITHUB_ENV
+    - name: Run container build script
+      run: bin/build_container_image
```

Comment:
Is this doing a build on pull requests in addition to pushes?

Reply:
Yes, it builds on every PR, but only pushes to the registry on merges

Reply:
I split it so that the credentials shouldn't even show up on PR run.